modules/microshift-install-system-requirements.adoc

```diff
@@ -8,15 +8,21 @@
 
 The following conditions must be met prior to installing {microshift-short}:
 
-* A compatible version of {op-system-base} or {op-system-ostree}.
+* A compatible version of {op-system-base-full}.For more information, see link:https://docs.redhat.com/en/documentation/red_hat_build_of_microshift/{ocp-version}/html/getting_ready_to_install_microshift/microshift-install-get-ready#get-ready-install-rhde-compatibility-table_microshift-install-get-ready[Compatibility table].
```

Comment:
This link gives me a 404 error, even when the version number correctly resolves. If this is a module that would only ever go in microshift docs, I think it would be more appropriate to just mention the page by name, and then use an xref in an additional resources section under the module include statement

Reply:
That is because 4.18 is not GA yet.